GoogleDataTransportCCTSupport/GoogleDataTransportCCTSupport/ProtoSupport/generate_cct_protos.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
I'm (obviously) unfamiliar with your project, so I'm not sure:
a) what platforms are being targeted. (Presumably iOS?)
b) build system (looks like there isn't one yet, but maybe xcode or cmake?)

FWIW, we hooked up similar functionality into our cmake build. We think hooking it into our xcode build would be possible too, but we haven't done that yet. (Implication: when our protos change, we need to run the cmake build before we can use those changes in our xcode build.) But prior to wiring it into cmake, we just manually regenerated the protos via a script similar to this one.

Reply:
a) iOS
b) Xcode, this will ship as a source Cocoapod

The protos are regenerated every time the xcodeproj is generated from the podspec. As this is part of the standard workflow of this library, I expect that the generated sources will remain up to date.